src/module.c

```diff
@@ -3082,7 +3082,7 @@ int DistAggregateCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc
   IndexSpec *sp = StrongRef_Get(spec_ref);
   if (!sp) {
     // Reply with error
-    return RedisModule_ReplyWithErrorFormat(ctx, "%s: no such index", idx);
+    return RedisModule_ReplyWithErrorFormat(ctx, "No index exists with provided name %s", idx);
```

Comment:
Rephrasing? Why?
If rephrasing consider this
```suggestion
    return RedisModule_ReplyWithErrorFormat(ctx, "No index exists with the provided name %s", idx);
```

Reply:
Redis takes the first string as the error code.
Don't want user data to be taken as an error code.
Will apply your suggestion.